var/spack/repos/builtin/packages/kaldi/package.py

```diff
@@ -47,19 +49,22 @@ class Kaldi(Package):    # Does not use Autotools
             description='build with CUDA')
 
     depends_on('blas')
-    depends_on('speex')
-    depends_on('openfst@1.6:')
     depends_on('cuda', when='+cuda')
     depends_on('sph2pipe', type='run')
     depends_on('sctk', type='run')
+    depends_on('speex', type='run')
+    depends_on('openfst@1.4.1-patch', when='@c024e8')
+    depends_on('openfst')
+
+    patch('openfst-1.4.1.patch', when='@c024e8')
 
     def install(self, spec, prefix):
-        configure_args = [
-            '--threaded-math',
-            '--speex-root=' + spec['speex'].prefix,
-            '--fst-root=' + spec['openfst'].prefix,
-            '--fst-version=' + str(spec['openfst'].version)
-        ]
+        configure_args = ['--fst-root=' + spec['openfst'].prefix]
+
+        if str(self.version) != 'c024e8':
```

Comment:
It's probably cleaner to use `spec.satisfies('@c024e8')` or `self.version == Version('c024e8')`. At least that's what other packages use.